server/server-app.ts

```diff
@@ -56,7 +56,7 @@ PersistenceInstance.hardRefresh$()
     startWith(true)
   )
   .subscribe(() => {
-    console.info('Data Hard-Refresh');
+    LOGGER.info('Data Hard-Refresh');
```

Comment:
oh wow you replaced a bunch of loggers - those could be its own PR (if you have the time for it - if not - no worries here :)

Reply:
By the end of it, I figured that 😆. It's just so we can handle pretty much all the logs properly, or otherwise we'd have a mix of JSON logs and plain text logs.

Maybe we can add something to `eslint` to deny any `console` calls in the `server/**` files?

Also, what happens to other calls to `console` outside the server package (that are shared between UI and server, if any)?
Perhaps we move the logger into a separate package that all packages depend on with a common interface. Then the UI forwards to console calls and the server forwards to the `NamedLogger`. What do you think?

Reply:
right before this PR there wasn't a need (yet, it was on my todo list yes) to have all logs done in a normed way

I can try to add some lints for those yeah, as for a shared way (since some libs are used between server and UI) I cant use the one from TsED ^^  soo we'd need some kind of shared logger 

all TsED Services/Controller get it by using dependency injection - all others would need some kind of interface which is set by each platform (server, web) - I don't expect that to happen fully today but the linting part will

Reply:
speaking of "Done" Logger Entry

wouldn't this one work? 

`'Server is ready`  ?